azkaban-common/src/test/java/azkaban/jobtype/JobTypeManagerTest.java

```diff
@@ -53,6 +59,20 @@ public JobTypeManagerTest() {
 
   @Before
   public void setUp() throws Exception {
+    // TODO: reallocf Remove injector creation/deletion when ProcessJob is fully guicified
+    final Props props = new Props();
+
+    final Injector injector = Guice.createInjector(new AbstractModule() {
```

Comment:
Yes, I am in favor of phased approach.

In this case, I replaced your Guice initiailization code with 
import static azkaban.test.Utils.initServiceProvider;

initServiceProvider();

And the tests in this file passed